README.md

```diff
@@ -102,8 +102,10 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Fairphone 2 - FP2
 - LG G3 - D855
 - LG Google Nexus 5 - hammerhead D820, D821
+- OnePlus One - bacon (use `lk2nd-appended-dtb.img`)
```

Comment:
Do you mean `lk2nd-msm8974-appended-dtb.img `?

Reply:
I guess, when built from source tree it's just called `lk2nd-appended-dtb.img` though, just differs for GH releases